logstash-output-nagios.gemspec

```diff
@@ -1,7 +1,7 @@
 Gem::Specification.new do |s|
 
   s.name            = 'logstash-output-nagios'
-  s.version         = '1.0.0'
+  s.version         = '1.1.0'
```

Comment:
Speaking about version numbers, please see https://github.com/logstash-plugins/logstash-input-zeromq/pull/8#issuecomment-135874348 for a long explanation. Happy to be educated, but kinda don't see your point either :smile_cat: